components/infra-proxy-service/README.md

```diff
@@ -1,3 +1,73 @@
-# Automate Infra Proxy Service
+# Infra Proxy Service
+
+Infra Proxy Service is an API for fetching and managing data of Chef Infra Server using Chef Infra Server API. Automate infra views consume these API to provides the capabilities of Chef Manage into Chef Automate in order to have single source of truth.
```

Comment:
Open for any other better infra service summary here?

Reply:
what you put sounds good to me!